code/ui/WeaponSelection.cs

```diff
@@ -162,7 +162,7 @@ public WeaponSlot(Panel parent, TTTWeapon weapon)
                 Parent = parent;
                 WeaponName = weapon.GetName();
 
-                _slotLabel = Add.Label(((int) weapon.WeaponType).ToString(), "slotlabel");
+                _slotLabel = Add.Label(((int)weapon.WeaponType).ToString(), "slotlabel");
```

Comment:
imo, there should be a space between type casting (explicit conversions) and the var

Reply:
https://github.com/TTTReborn/ttt-reborn/blob/fea10d705fbdb9b9214753681e98dabd712e922a/.editorconfig#L146

my formatter only did what was specified in the .editorconfig :P
i can change this if the team agrees

Reply:
i would also change it